lib/vtt.js

```diff
@@ -1204,8 +1204,17 @@
     })();
   };
 
-  WebVTT.Parser = function(window, decoder) {
+  WebVTT.Parser = function(window, vttjs, decoder) {
```

Comment:
I just went with a name consistent with the object you'd be passing in. Since we're now exporting the vttjs shims into a `vttjs` object, you can do something like:
```js
var parser = new WebVTT.Parser(window, vttjs, WebVTT.StringDecoder)
```